public/index.ts

```diff
@@ -1082,13 +1111,74 @@ function createTestVolume() {
   };
 }
 
+function createLoader(type: string): IVolumeLoader {
+  switch (type) {
+    case "opencell":
+      return new OpenCellLoader();
+    case "omezarr":
+      return new OMEZarrLoader();
+    case "ometiff":
+      return new TiffLoader();
+    // case "procedural":
+    //   return new RawVolumeLoader();
```

Comment:
the above is a hint to imply that we could make a loader that just accepts a TypedArray with the volume data but currently the creation is so simple it seemed overkill.